pkg/deploy/ec2/endpoint_service_manager.go

```diff
@@ -0,0 +1,265 @@
+		}
+		resLBArnsRaw = append(resLBArnsRaw, arn)
+	}
+
+	if err := m.updateSDKVPCEndpointServiceWithTags(ctx, resES, sdkES); err != nil {
+		return ec2model.VPCEndpointServiceStatus{}, err
+	}
+
+	addLBArns, _, removeLBArns := algorithm.DiffStringSlice(resLBArnsRaw, sdkES.NetworkLoadBalancerArns)
+	// The API call expects these to be nil if no changes are required.  An empty list returns an error
+	if len(addLBArns) == 0 {
+		addLBArns = nil
+	}
+	if len(removeLBArns) == 0 {
+		removeLBArns = nil
+	}
+
+	var acceptanceRequired *bool
+	if resES.Spec.AcceptanceRequired != nil && *resES.Spec.AcceptanceRequired != sdkES.AcceptanceRequired {
+		acceptanceRequired = resES.Spec.AcceptanceRequired
+	}
+
+	var privateDNSName *string
+	var removePrivateDNSName *bool
+	if resES.Spec.PrivateDNSName == nil && sdkES.PrivateDNSName != nil {
+		removePrivateDNSName = awssdk.Bool(true)
+	} else if resES.Spec.PrivateDNSName != sdkES.PrivateDNSName {
+		privateDNSName = resES.Spec.PrivateDNSName
+	}
+
+	if len(addLBArns) > 0 || len(removeLBArns) > 0 || acceptanceRequired != nil || privateDNSName != nil || removePrivateDNSName != nil {
+
+		serviceId := &sdkES.ServiceID
+
+		m.logger.Info(
+			"Updating VPCEndpointService",
+			"addLBArns", addLBArns,
+			"removeLBArns", removeLBArns,
+			"acceptanceRequired", acceptanceRequired,
+			"privateDNSName", privateDNSName,
+			"removePrivateDNSName", removePrivateDNSName,
+			"serviceId", serviceId,
+		)
+
+		req := ec2sdk.ModifyVpcEndpointServiceConfigurationInput{
+			AcceptanceRequired:            acceptanceRequired,
+			AddNetworkLoadBalancerArns:    addLBArns,
+			RemoveNetworkLoadBalancerArns: removeLBArns,
+			PrivateDnsName:                privateDNSName,
+			RemovePrivateDnsName:          removePrivateDNSName,
+			ServiceId:                     serviceId,
+		}
+
+		_, err := m.ec2Client.ModifyVpcEndpointServiceConfigurationWithContext(ctx, &req)
+		if err != nil {
+			return ec2model.VPCEndpointServiceStatus{}, err
+		}
+	} else {
+		m.logger.Info(
+			"Not updating VPCEndpointService",
+		)
+	}
+
+	return ec2model.VPCEndpointServiceStatus{
+		ServiceID: sdkES.ServiceID,
+	}, nil
+}
+
+func (m *defaultEndpointServiceManager) Delete(ctx context.Context, sdkES networking.VPCEndpointServiceInfo) error {
+	req := &ec2sdk.DeleteVpcEndpointServiceConfigurationsInput{
+		ServiceIds: awssdk.StringSlice(
+			[]string{sdkES.ServiceID},
+		),
+	}
+
+	m.logger.Info("deleting VPCEndpointService",
+		"serviceId", sdkES.ServiceID)
+	if _, err := m.ec2Client.DeleteVpcEndpointServiceConfigurationsWithContext(ctx, req); err != nil {
+		return errors.Wrap(err, "failed to delete VPCEndpointService")
+	}
+	m.logger.Info("deleted VPCEndpointService",
+		"serviceId", sdkES.ServiceID)
+
+	return nil
+}
+
+func (m *defaultEndpointServiceManager) ReconcilePermissions(ctx context.Context, permissions *ec2model.VPCEndpointServicePermissions) error {
+	m.logger.Info("Reconciling Permissions")
```

Comment:
The pattern through the rest of the codebase is lower case logging. Do we want to deviate from that here?